Index: producer/main.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+># producer/main.py\r\n\r\nfrom fastapi import FastAPI\r\nfrom producer.router import router\r\nimport uvicorn\r\n\r\ndef create_app() -> FastAPI:\r\n    event_producer_app = FastAPI(title=\"Event Producer Service\")\r\n    event_producer_app.include_router(router.router)\r\n    return event_producer_app\r\n\r\napp = create_app()\r\n\r\nif __name__ == \"__main__\":\r\n    uvicorn.run(\"producer.main:app\", host=\"0.0.0.0\", port=8000, reload=True)\r\n
===================================================================
diff --git a/producer/main.py b/producer/main.py
--- a/producer/main.py	(revision 08a6553e73076e921d4b75fd6bb5af34330dab59)
+++ b/producer/main.py	(date 1747569697230)
@@ -1,5 +1,3 @@
-# producer/main.py
-
 from fastapi import FastAPI
 from producer.router import router
 import uvicorn
